src/rust/engine/Cargo.toml

```diff
@@ -39,6 +39,7 @@ members = [
   "resettable",
   "testutil",
   "testutil/mock",
+  "ui"
```

Comment:
This should also be in `default-members` I think, as it doesn't have any system requirements.

Reply:
fixed